.github/auto_request_review.yml

```diff
@@ -0,0 +1,23 @@
+reviewers:
+  defaults:
+    - rollkit
+  groups:
+    rollkit:
+      - Manav-Aggarwal
+      - tzdybal
+      - gupadhyaya
+      - tuxcanfly
+      - MSevey
+      - yarikbratashchuk
+      - jcstein
+files:
+  "**":
+    - rollkit
+  ".github/**":
+    - MSevey
+    - rollkit
+options:
+  ignore_draft: true
+  ignored_keywords:
+    - WIP
+  number_of_reviewers: 3
```

Comment:
does this randomly rotate reviewers @MSevey?

Reply:
yes, it will randomly select 3 from the list. 
If you want to always be assigned I can update it to be that.

Reply:
I don't mind being assigned to all, with others being rotated. if it gets to a point where I hit review fatigue, I'll open PR to change it?

Reply:
sounds good. I think the following updates should make it so you are always assigned.
If we see PRs where that is not the case we can look at other options.